candi/openapi/node_group.yaml

```diff
@@ -204,6 +204,8 @@ spec:
                         Container runtime type.
 
                         Value `defaultCRI` from the initial cluster configration (`cluster-configuration.yaml` parameter from the `d8-cluster-configuration` secret in the `kube-system` namespace) is used if not specified.
+
+                        > **Note!** the `Docker` is deprecated.
```

Comment:
Hm. Don't we have to remove the `Docker` value from enum.
With a release requirements check we should't have NodeGroups with a Docker value inside a cluster